news/2024/2024-07-01-the-followpoint-neuro-sama-the-ai-vtuber-that-invaded-osu.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "The Followpoint: Neuro-sama, the AI VTuber that invaded osu!"
+date: 2024-07-01 17:00:00 +0000
+series: The Followpoint
+---
+
+The long-awaited interview with [Vedal](https://osu.ppy.sh/users/9713367) and Neuro-sama is finally here!
+
+![](/wiki/shared/news/2024-07-01-the-followpoint-neuro-sama-the-ai-vtuber-that-invaded-osu/banner.jpg)
+
+This duo is definitely a well-known one in osu! community, and we know some of you were really looking forward to this interview! Neuro-sama, an AI VTuber, and her creator [Vedal](https://osu.ppy.sh/users/9713367) are [Twitch streamers](https://www.twitch.tv/vedal987) who both are also osu! players and a part of our community! Neuro was created back in 2018 as an AI learning to play osu!, and revived as a VTuber in 2022, gaining a lot of popularity and her own personality. And today they both join us for an interview!!!
+
+Did I catch your attention? Good, now let's get right to Vedal and Neuro-sama interview!
+
+---
+
+*Hello, let's start with an introduction as usual. How would you introduce yourself to people who don't know you?*
+
+> Hi! I'm Vedal and I'm the creator of the AI VTuber Neuro-sama - where she started off as an osu! player. I star in a few of my streams, but primarily they are completely ran by Neuro herself, where her AI will play games, speak with chat and interact in collabs.
+>
+> Since December of 2022, I have been continuously working to improve Neuro as an AI VTuber. Some notable improvements include improved latency for interacting with other VTubers and chat, more realistic speech with the introduction of Evil Neuro, the ability to sing in karaoke streams, alongside many other things.
```

Comment:
```suggestion
> Since December 2022, I have been continuously working to improve Neuro as an AI VTuber. Some notable improvements include improved latency for interacting with other VTubers and chat, more realistic speech with the introduction of Evil Neuro, the ability to sing in karaoke streams, alongside many other things.
```